src/com/core/tictactoe/Game.java

```diff
@@ -0,0 +1,43 @@
+package com.core.tictactoe;
+
+public class Game {
+
+    Board board;
+    Player active;
+    Player passive;
+    CommandLineUI commandLineUI;
+
+    Game (CommandLineUI commandLineUI) {
+        this.board = new Board();
+        this.active = new Player("X");
+        this.passive = new Player("Y");
```

Comment:
These players being constructed here are okay for now, but what if you needed different types of players depending on what game type was selected?

Reply:
Then I'll sort out the type of game (and so types of players) in the pre-game phase (in GamesController class) and then pass the players to the game using dependency injection.